components/public-api/gitpod/v1/verification.proto

```diff
@@ -0,0 +1,155 @@
+syntax = "proto3";
+
+package gitpod.v1;
+
+import "gitpod/v1/pagination.proto";
+import "gitpod/v1/sorting.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/v1";
+
+service VerificationService {
+  // SendPhoneNumberVerificationToken sends a verification token to the
+  // specified phone number.
+  rpc SendPhoneNumberVerificationToken(SendPhoneNumberVerificationTokenRequest) returns (SendPhoneNumberVerificationTokenResponse) {}
+
+  // VerifyPhoneNumberVerificationToken verifies the specified verification
+  // token.
+  rpc VerifyPhoneNumberVerificationToken(VerifyPhoneNumberVerificationTokenRequest) returns (VerifyPhoneNumberVerificationTokenResponse) {}
+
+  // ListBlockedRepositories lists blocked repositories.
+  rpc ListBlockedRepositories(ListBlockedRepositoriesRequest) returns (ListBlockedRepositoriesResponse) {}
```

Comment:
@mustard-mh I chatted with @AlexTugarev and @geropl about ti, management of blocked repositories and email domains should go to a separate InstallationService. I updated the table.

Reply:
@jeanp413  new service name will be `InstallationService` ☝️